api/applications/serializers.py

```diff
@@ -0,0 +1,60 @@
+from django.core.validators import URLValidator
+from rest_framework import serializers as ser
+
+from website.models import ApiOAuth2Application
+
+from api.base.serializers import JSONAPISerializer, LinksField
+
+
+class ApiOAuth2ApplicationSerializer(JSONAPISerializer):
+    """Serialize data about a registered OAuth2 application"""
+    client_id = ser.CharField(help_text="The client ID for this application (automatically generated)",
+                              read_only=True)
+
+    client_secret = ser.CharField(help_text="The client secret for this application (automatically generated)",
+                                  read_only=True)  # TODO: May change this later
+
+    owner = ser.CharField(help_text="The id of the user who owns this application",
+                          read_only=True,  # Don't let user register an application in someone else's name
+                          source='owner._id'
+                          )
```

Comment:
Very minor style critique - this field has its closing brace on a line by itself, the others do not.